apps/test/unit/templates/studioHomepages/StudentHomepageTest.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { assert, expect } from 'chai';
+import { shallow } from 'enzyme';
+import StudentHomepage from '@cdo/apps/templates/studioHomepages/StudentHomepage';
+
+const sections = [
```

Comment:
Can we deduplicate the test sections and courses we use here with some of our other `studioHomepages` tests?